config/routes.rb

```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
 
+  Healthcheck.routes(self)
```

Comment:
🤔 Trying to figure out how this even got imported into this file ❓ Ruby is a mysterious mystery 

Might be helpful to have a comment pointing to the healthcheck config file. Or, more specifically something indicating that the healthcheck route is `/healthcheck`.

Reply:
➕ to Ruby being a mysterious mystery 😂

Reply:
I know this is poor form, but I addressed this in [another PR I had open](https://github.com/WPMedia/klaxon/pull/4/files#r1058662043)!